packages/variants/src/variantThemeType.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Const enum for variant theme types.
```

Comment:
Not sure where cliff's comment went, but +1 on documenting each of these specifically (instead of in getSoftVariant, etc functions where i have them documented now)

Reply:
I'm not familiar with the JSDoc thing mentioned above. How can I find out more? Can I get more specific direction or at least can you give me a file path to look for?

Reply:
@bennettclark JSDoc refers to the `/** */` documentation comments. So the ask here is to document each of these enums.


There are ample examples in the main OUFR library at: https://github.com/OfficeDev/office-ui-fabric-react/blob/1ef7812da2f994edce867acdd6f58e82688c816d/packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.types.ts#L453

Reply:
@cliffkoh @phkuo Thanks for the clarification! Added comments. Please let me know if anything further is needed!